wallet/how-to/manage-networks/add-network.md

```diff
@@ -5,6 +5,10 @@ sidebar_position: 2
 
 # Add a network
 
+:::note
+See the [Connect to Hardhat Network](../how-to/run-devnet.md) documentation to run a development network using Hardhat and connect MetaMask to it.
+:::
+
```

Comment:
I would make the note less prominent - maybe move to after the screenshots. And update it to something like the following for simplicity:

```
:::note
You can also [add a Hardhat development network to MetaMask](../how-to/run-devnet.md).
:::
```

Reply:
Can the messaging be a bit more general? 

```
Looking for information on how to add a local development network to MetaMask, such as Hardhat? [Learn how to connect a development network.](...)
```